lib/core/presentation/pages/event/event_detail_page/helper/event_features_helper.dart

```diff
@@ -0,0 +1,245 @@
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/core/utils/modal_utils.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/router/app_router.gr.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_vibrate/flutter_vibrate.dart';
+
+class FeatureItem {
```

Comment:
As this only used in this file, recommend to add _ as a prefix , so that prevent duplicated import in the future, can we rename to _NavigationItem